database/seeders/PermissionSeeder.php

```diff
@@ -129,6 +129,11 @@ public function run(): void
                 'question-bank.read',
                 'question-bank.update',
                 'question-bank.delete',
+
+                'widgets.create',
+                'widgets.read',
+                'widgets.update',
+                'widgets.delete',
```

Comment:
These won't be added fella. As they're not run more than once in a live system. You'll need a migration or something to add these to an established system